### PR TITLE
chore: Add ClerkProvider properties

### PR DESCRIPTION
**Pull request by LekoArts**
> [!IMPORTANT]
> 🔎 Previews:
>
> -

While working on https://github.com/clerk/javascript/pull/4098 I saw that some properties were missing. So I added them.

### CODE REVIEW

File: docs/components/clerk-provider.mdx [hunk 1]
```diff
@@ -204,7 +204,7 @@ The `<ClerkProvider>` component must be added to your React entrypoint.
   - `signInUrl`
   - `string`
 
-  This URL will be used for any redirects that might happen and needs to point to your primary application on the client-side. This option is optional for production instances and required for development instances. It's recommended to use [the environment variable](/docs/deployments/clerk-environment-variables#sign-in-and-sign-up-redirects) instead.
+  This URL will be used for any redirects that might happen and needs to point to your primary application on the client-side. This option is optional for production instances. It's required for development instances if you a use satellite application. It's recommended to use [the environment variable](/docs/deployments/clerk-environment-variables#sign-in-and-sign-up-redirects) instead.
```
Comment (alexisintech) [inline docs/components/clerk-provider.mdx:207]:
```suggestion
  This URL will be used for any redirects that might happen and needs to point to your primary application on the client-side. This option is optional for production instances. It's required for development instances if you use a satellite application. It's recommended to use [the environment variable](/docs/deployments/clerk-environment-variables#sign-in-and-sign-up-redirects) instead.
```

Comment (alexisintech) [reply docs/components/clerk-provider.mdx:207]: Is it required to be set in the satellite application, or the primary application ?

Comment (LekoArts) [reply docs/components/clerk-provider.mdx:207]: In the satellite application

File: docs/components/clerk-provider.mdx [hunk 2]
```diff
@@ -204,7 +204,7 @@ The `<ClerkProvider>` component must be added to your React entrypoint.
   - `signInUrl`
   - `string`
 
-  This URL will be used for any redirects that might happen and needs to point to your primary application on the client-side. This option is optional for production instances and required for development instances. It's recommended to use [the environment variable](/docs/deployments/clerk-environment-variables#sign-in-and-sign-up-redirects) instead.
+  This URL will be used for any redirects that might happen and needs to point to your primary application on the client-side. This option is optional for production instances. It's required for development instances if you use a satellite application. It's recommended to use [the environment variable](/docs/deployments/clerk-environment-variables#sign-in-and-sign-up-redirects) instead.
```
Comment (alexisintech) [inline docs/components/clerk-provider.mdx:207]:
```suggestion
  This URL will be used for any redirects that might happen and needs to point to your primary application on the client-side. This option is optional for production instances. **It is required to be set for a satellite application in a development instance.** It's recommended to use [the environment variable](/docs/deployments/clerk-environment-variables#sign-in-and-sign-up-redirects) instead.
```

**Review by alexisintech — APPROVED** [review]

**Pull request merged**